ckan/templates-midnight-blue/snippets/language_selector.html

```diff
@@ -1,5 +1,5 @@
 {% set current_lang = request.environ.CKAN_LANG %}
-<form class="lang-select" action="{% url_for 'util.internal_redirect' %}" data-module="select-switch" method="POST">
+<form class="lang-select" action="{% url_for 'util.internal_redirect' %}" data-module="select-switch" method="GET">
   {{ h.csrf_input() }}
```

Comment:
we can remove this now, right?

Reply:
prefer to leave it explicit as its against the grain of most other forms.
also missed removing the {{ h.csrf_input() }} on blue.